src/jsonld/localBusiness.tsx

```diff
@@ -27,14 +27,14 @@ type OpeningHoursSpecification = {
 
 export interface LocalBusinessJsonLdProps {
   type: string;
-  id: string;
+  id?: string;
```

Comment:
```suggestion
  id: string;
```

So although we will check for this and not render if not passed in, it is required according to the spec so for anyone that uses Typescript we will signal this.